sympy/calculus/singularities.py

```diff
@@ -197,6 +312,13 @@ def is_monotonic(expression, interval=S.Reals, symbol=None):
     """
     Return whether the function is monotonic in the given interval.
 
+    expression: Expr
+        The target function which is being checked.
+    interval: Symbol
+        The range of values in which we are testing, defaults to all reals.
+    symbol: Symbol
```

Comment:
One thing to check is whether this strictly has to be a Symbol. Often a Function might work or even sometimes an expression. I don't know how restrictive these functions are but many sympy functions aren't as restrictive. So try to check the restrictions of input type and then you can do "Symbol or Function" or "sympyfiable" if sympify is called internal to the function.